lighthouse-cli/performance-experiment/report/partials/left-nav.html

```diff
@@ -0,0 +1,70 @@
+<style>
+:root[data-report-context="perf-x"] .menu__link {
+  padding: 0 35px;
+}
+
+.menu__report-tab {
+  padding: 3px 13px;
+  color: #777;
+  border-top: 1px solid #EBEBEB;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  text-decoration: none;
+  height: 55px;
+}
+
+.menu__report-tab:hover {
+  background-color: #448aff;
+  color: #FFF;
+}
+
+.menu__report-tab__url {
+  font-size: 15px;
+  font-weight: bold;
+  text-overflow: ellipsis;
+  line-height: 30px;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.menu__report-tab__time {
+  font-size: 11px;
+  line-height: 11px;
+  padding-left: 2px;
+}
+</style>
+
+{{#createRelatedReportsContext}}
```

Comment:
i think we can do this partial better.

one approach: single loop through the reports. check if the reportId matches the requestedID and if it does, tweak the `menu__report-tab` link.  maybe generate the `menu__nav`'s for all the reports, but display:none the others with some classes/css

probably some better ways than that, but let's DRY this up some.